versioned_hdf5/subchunk_map.py

```diff
@@ -270,12 +271,25 @@ def index_chunk_mappers(
 
     d: hsize_t
     n: hsize_t
+    fancy_count = 0
     mappers = []
 
     # Process the prefix of the axes which idx selects on
     for i, d, n in zip(idx.args, shape[:idx_len], chunk_size[:idx_len]):
-        i = i.reduce((d,))
-        mappers.append(_index_to_mapper(i.raw, d, n))
+        if isinstance(i, ellipsis):  # ndindex wrapper for Ellipsis; reduces erratically
+            raise NotImplementedError("Ellipsis not supported")
```

Comment:
Adding support for ellipsis (and just as importantly, adding unit tests) is substantially more involved and I'd rather leave it as a follow-up PR after #386